src/devices/thermopro_tp28b.c

```diff
@@ -0,0 +1,171 @@
+/** @file
+    ThermoPro TP28b Super Long Range Wireless Meat Thermometer for Smoker BBQ Grill.
+
+    Copyright (C) 2024 Josh Pearce <joshua.pearce@gmail.com>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+#include "decoder.h"
+
+/**
```

Comment:
When using helper functions this line needs to be:
`/** @fn int thermopro_tx2c_decode(r_device *decoder, bitbuffer_t *bitbuffer)`

Reply:
Thanks. And I changed the function name 😄